build.gradle

```diff
@@ -18,7 +18,7 @@ ext {
   baseVersion = "2.4"
   snapshotVersion = true
   milestone = 0
-  javaVersions = [8, 11, 17, 21] // ensure that latest version is actually build on GH actions and added to gradle.properties, otherwise no docs get published
+  javaVersions = [8, 11, 17, 22] // ensure that latest version is actually build on GH actions and added to gradle.properties, otherwise no docs get published
```

Comment:
We don't need to add it here, you can already build with any java version, there are just some tasks that will complain.

Reply:
I remember docs were not released on version mismatch. Have you already improved it in the mean time? If yes, how would you like to keep it?

Reply:
That is true, but with my proposed changes 22 would only be used to test and we'd stick with 21 for the release.

Reply:
Sure, changed.